src/components/Footer.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import {
+  Grid,
+  Link,
+  Stack,
+  ThemeProvider,
+  Typography,
+  createTheme,
+} from "@mui/material";
+document.body.style.color = "black";
+
+const Footer = () => {
+  let theme = createTheme({
+    components: {
+      MuiLink: {
+        styleOverrides: {
+          root: {
+            textDecoration: "none",
+            color: "black",
+            fontSize: "11px",
+            fontWeight: "600",
+            "&:hover": {
+              color: "#1db272",
+              transition: "200ms ease-in",
+            },
+          },
+        },
+      },
+    },
+    typography: {
+      h1: {
+        fontWeight: "600",
+        fontSize: "20px",
+        lineHeight: "20px",
+      },
+      h2: {
+        textDecoration: "underline",
+        fontSize: "13px",
+        fontWeight: "600",
+      },
+      h3: {
+        fontSize: "13px",
+        fontWeight: "400",
+      },
+    },
+  });
+
+  /**
+   * @TODO : add actual links to the link elements (https://github.com/LetsMesh/Site/issues/191)
+   * */
```

Comment:
Ideally this `TODO` would go directly above the associated code. If we switch to HTML-style comments, we can move this to just above line 79.